src/Raven.Server/Documents/PeriodicBackup/Aws/RavenAwsS3Client.cs

```diff
@@ -245,16 +248,21 @@ private async Task<string> UploadPart(Stream baseStream, HttpClient client, stri
 
             // wait for one second before trying again to send the request
             // maybe there was a network issue?
-            await Task.Delay(1000);
+            Thread.Sleep(1000);
```

Comment:
Let's use the database shutdown cancellation token here.

Reply:
done